devito/function.py

```diff
@@ -920,11 +929,13 @@ def coefficients(self):
     @property
     def point_symbols(self):
         """Symbol for coordinate value in each dimension of the point"""
-        return sympy.symbols('px, py, pz')
+        return tuple(sympy.symbols('p%s' % d) for d in self.grid.dimensions)
```

Comment:
I suggest we use our own ``Symbol`` from ``types.py``

Reply:
These are not Devito object, pure sympy symbols fro substitutions. Our own Symbol are not recognized by `subs`.